iOS/Source/MainView.swift

```diff
@@ -201,8 +201,8 @@ class MainView: UIView {
 
             UIView.animate(withDuration: 0.4) {
                 self.backgroundColor = backgroundColor
-                self.sunView.updateInterface(withBackgroundColor: backgroundColor, textColor: textColor, andPercentageInDay: percentageInDay, sunPhase: sunPhase)
                 self.sunView.update(for: location)
+                self.sunView.updateInterface(withBackgroundColor: backgroundColor, textColor: textColor, andPercentageInDay: percentageInDay, sunPhase: sunPhase)
```

Comment:
Nice, I think merging these methods might be good to reduce side effects of using them in the incorrect order,